Backend/CarRentalApp/CarRentalApp/Models/DTOs/Requests/UserLoginDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace CarRentalApp.Models.DTOs.Requests
+{
+    public class UserLoginDTO
+    {
+        [Required]
+        public string Username { get; set; }
+
+        [Required]
+        [RegularExpression(
+            @"^" +
+            "(?=.*[A-Z])" + // At least one UpperCase
+            "(?=.*[0-9])" + // At least one Digit
+            "(?=.*[a-z])" + // At least one LowerCase
+            ".{5,25}" +     // Length between 5 and 25
+            "$",
+         ErrorMessage = "Incorrect format")]
```

Comment:
How would user know what exactly is incorrect? Also why checking this when you are trying to login, i.e. password already set for this user?